lib/meshchat/configuration/hash_file.rb

```diff
@@ -23,11 +23,9 @@ def load
         f = read_file
         begin
           self._hash = JSON.parse(f)
-        rescue => e
-          Display.alert e.message
-          self._hash = default_settings
```

Comment:
TODO: maybe I should add this back. I only really needed to remove the calls to display.